cloud/azure/README.md

```diff
@@ -1,7 +1,34 @@
-# Azure Terraform 
+# Azure Terraform Setup
 
```

Comment:
thought: I think this line will work better as an intro at the top of the file than in the subsection it was in.
```suggestion
We use a remote backend to store state information to avoid conflicts when multiple collaborators are updating resources managed by Terraform. Follow the steps outlined in this doc to enable collaborators to access the remote backend and shared state on their machines.
```